tox.ini

```diff
@@ -3,19 +3,19 @@ envlist = min,py39,py310,py311
 skip_missing_interpreters = True
 
 [testenv]
-extras =
-    tests
-    validation
+deps =
+    pytest
+    pytest-cov
+    pytest-mock
 commands = pytest -s --cov=spidermon --cov-report= {posargs:tests}
 
 [testenv:min]
 basepython = python3.8
-deps =
-    jsonschema[format]==3.2.0
```

Comment:
Why is this line removed? I think the intention was to pin to a min package version, and test against it; in other environments a lates vertsion is going to be installed. It could make sense to pin other packages here as well (like scrapy), if a minimum version requirement is known.

Reply:
My bad, I saw it was already pinned in `install_requires` but I didn't noticed that it was `>=` instead of `==`